GNNGraphs/ext/GNNGraphsCUDAExt.jl

```diff
@@ -20,6 +21,16 @@ GNNGraphs.dense_zeros_like(a::CUMAT_T, T::Type, sz = size(a)) = CUDA.zeros(T, sz
 
 GNNGraphs.iscuarray(x::AnyCuArray) = true
 
+function GNNGraphs.binarize(Mat::CUSPARSE.CuSparseMatrixCSC)
+    bin_vals = fill!(similar(nonzeros(Mat), Bool), true)
+    return CUSPARSE.CuSparseMatrixCSC(Mat.colPtr, rowvals(Mat), bin_vals, size(Mat))
+end
```

Comment:
```suggestion
```
This could be removed in favor of a single method with signature `binarize(Mat::CUSPARSE.CuSparseMatrixCSC, T::DataType = Bool). Same for the main method in GNNGraphs I guess.